src/background.js

```diff
@@ -181,22 +183,22 @@ function createWindow() {
     }
   })
 
-  if (!isDevelopment) {
-    // Disable shortcuts defining a hidden menu and binding the shortcut we
-    // want to disable to an option
-    const menu = Menu.buildFromTemplate([
-      {
-        label: 'Menu',
-        submenu: [
-          { label: 'Reload', accelerator: 'CmdOrCtrl+R', click: () => {} },
-          { label: 'ZoomOut', accelerator: 'CmdOrCtrl+-', click: () => {} },
-          { label: 'ZoomIn', accelerator: 'CmdOrCtrl+Plus', click: () => {} },
-        ],
-      },
-    ])
-
-    Menu.setApplicationMenu(menu)
-  }
+  // if (!isDevelopment) {
+  //   // Disable shortcuts defining a hidden menu and binding the shortcut we
+  //   // want to disable to an option
+  //   const menu = Menu.buildFromTemplate([
+  //     {
+  //       label: 'Menu',
+  //       submenu: [
+  //         { label: 'Reload', accelerator: 'CmdOrCtrl+R', click: () => {} },
+  //         { label: 'ZoomOut', accelerator: 'CmdOrCtrl+-', click: () => {} },
+  //         { label: 'ZoomIn', accelerator: 'CmdOrCtrl+Plus', click: () => {} },
+  //       ],
+  //     },
+  //   ])
+
+  //   Menu.setApplicationMenu(menu)
+  // }
```

Comment:
Is this within the scope of this PR?

Reply:
AFAIK, this should be a draft and this is useful to debug the build